pkg/ibmvpc/parser.go

```diff
@@ -1655,7 +1655,7 @@ func getLoadBalancerIPs(vpcConfig *vpcmodel.VPCConfig,
 // also update vpcConfig & subnet with the result
 func createPrivateIP(name, id, address, publicAddress string,
 	vpc vpcmodel.VPCResourceIntf, loadBalancer *LoadBalancer, vpcConfig *vpcmodel.VPCConfig,
-	original bool, subnet *Subnet) (*PrivateIP, error) {
+	original bool, block *ipblock.IPBlock, subnet *Subnet) (*PrivateIP, error) {
```

Comment:
The privateIP per block is not new to this PR. Why this change now?

Reply:
since we add it to the private ip name